### PR TITLE
Improve ambassador page

### PR DESCRIPTION
**Pull request by tsteenbe**
See individual commits for details.

### CODE REVIEW

**Comment by tsteenbe** [comment]
@anajsana  I did not add links for Jose Rodrigez Roa and Li Jiansheng as I could not find their GitHub profile or other page I could link to :-(

File: content/en/ambassadors.md [hunk 1]
```diff
@@ -17,7 +16,22 @@ The TODO OSPO Ambassador Program encompasses a group of community leaders who pr
...
-## Requirements
+## Meet our Ambassadors!
+
+| Name | Organization | Location | Local Efforts
+| --- | --- | --- | --- |
+| [Angel Ramirez](https://github.com/ar4mirez) | Cuemby | LATAM (Mexico 🇲🇽, Colombia 🇨🇴) | [OSPO Local Meetup LATAM](https://community.linuxfoundation.org/ospo-local-meetup-latam-spanish-speaking/) |
+| [Gergely Csatári](https://github.com/CsatariGergely) | Nokia | Finland 🇫🇮 | [OSPO Local Meetup Helsinki](https://community.linuxfoundation.org/ospo-local-meetup-helsinki/) |
+| [Hitomi Mizugaki](https://github.com/hit0mim)| Cuemby | United States (Florida 🇺🇸) | OSPO Local Meetup Florida |
+| Jose Rodrigez Roa | Transformente | LATAM (Mexico 🇲🇽, Colombia 🇨🇴, Ecuador 🇪🇨, Chile 🇨🇱) | [OSPO Local Meetup LATAM](https://community.linuxfoundation.org/ospo-local-meetup-latam-spanish-speaking/) |
+| Li Jiansheng | LF APAC | China 🇨🇳 | OSPO Local Meetup China |
```
Comment (anajsana) [inline content/en/ambassadors.md:27]:
```suggestion
| [Li Jiansheng](https://www.linkedin.com/in/lijiansheng/) | LF APAC | China 🇨🇳 | OSPO Local Meetup China |
```

File: content/en/ambassadors.md [hunk 2]
```diff
@@ -17,7 +16,22 @@ The TODO OSPO Ambassador Program encompasses a group of community leaders who pr
...
 
-## Requirements
+## Meet our Ambassadors!
+
+| Name | Organization | Location | Local Efforts
+| --- | --- | --- | --- |
+| [Angel Ramirez](https://github.com/ar4mirez) | Cuemby | LATAM (Mexico 🇲🇽, Colombia 🇨🇴) | [OSPO Local Meetup LATAM](https://community.linuxfoundation.org/ospo-local-meetup-latam-spanish-speaking/) |
+| [Gergely Csatári](https://github.com/CsatariGergely) | Nokia | Finland 🇫🇮 | [OSPO Local Meetup Helsinki](https://community.linuxfoundation.org/ospo-local-meetup-helsinki/) |
+| [Hitomi Mizugaki](https://github.com/hit0mim)| Cuemby | United States (Florida 🇺🇸) | OSPO Local Meetup Florida |
+| Jose Rodrigez Roa | Transformente | LATAM (Mexico 🇲🇽, Colombia 🇨🇴, Ecuador 🇪🇨, Chile 🇨🇱) | [OSPO Local Meetup LATAM](https://community.linuxfoundation.org/ospo-local-meetup-latam-spanish-speaking/) |
```
Comment (anajsana) [inline content/en/ambassadors.md:26]:
```suggestion
| [Jose Rodrigez Roa](https://www.linkedin.com/in/joserodriguezroa/) | Transformente | LATAM (Mexico 🇲🇽, Colombia 🇨🇴, Ecuador 🇪🇨, Chile 🇨🇱) | [OSPO Local Meetup LATAM](https://community.linuxfoundation.org/ospo-local-meetup-latam-spanish-speaking/) |
```

**Review by anajsana — COMMENTED** [review]
I've suggested adding the Linkedin profiles from the two missing contacts

File: content/en/ambassadors.md (hunk 2 above)
Comment (tsteenbe) [reply content/en/ambassadors.md]: Updated my commit to include Jose's linkedin.

**Review by anajsana — APPROVED** [review]
LGTM! Thanks for improving this page!

**Pull request merged**